### PR TITLE
fixed no content error

### PR DESCRIPTION
**Pull request by malavikakoppula**
**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

Showing error right after adding block to pb. 

## Jira Ticket

- [TMEDIA-745](https://arcpublishing.atlassian.net/browse/TMEDIA-745)

## Acceptance Criteria

[_copy from ticket_](https://arcpublishing.atlassian.net/browse/TMEDIA-745?focusedCommentId=780135)

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

**Review by brentmiller1973 — DISMISSED** [review]

**Comment by matthewroach** [comment]
@malavikakoppula I am still getting the error locally when adding this block within PageBuilder

File: blocks/large-manual-promo-block/features/large-manual-promo/default.jsx [hunk 1]
```diff
@@ -65,20 +65,24 @@ const LargeManualPromo = ({ customFields }) => {
...
+						openInNewTab={newTab}
+						onClick={registerSuccessEvent}
+					>
+						{headline}
+					</Conditional>
+				</Heading>
+			);
+		}
+		return null;
+	};
```
Comment (nschubach) [inline blocks/large-manual-promo-block/features/large-manual-promo/default.jsx]: This change doesn't change any logic, it's just a rewrite of what was there  The actual fix would be:
```suggestion
	const PromoHeading = () =>
		showHeadline && headline ? (
			<Heading>
				<Conditional
					component={Link}
					condition={linkURL}
					href={formatURL(linkURL)}
					openInNewTab={newTab}
					onClick={registerSuccessEvent}
				>
					{headline}
				</Conditional>
			</Heading>
		) : null;
```

Wouldn't it?

**Review by nschubach — CHANGES_REQUESTED** [review]
Need to check the `headline` content to fix that issue.

**Comment by matthewroach** [comment]
@malavikakoppula - I pushed a small update to address the issue still being seen within PageBuilder. Adding a block to the page in the first instance PageBuilder will not set these properties and they come through as `undefined` within the component. I updated the code to add check against the property along with the show logic

**Review by matthewroach — APPROVED** [review]

**Pull request merged**